libnavigation-base/src/main/java/com/mapbox/navigation/base/route/model/WalkingOptionsNavigation.kt

```diff
@@ -0,0 +1,8 @@
+package com.mapbox.navigation.base.route.model
+
+import com.google.gson.annotations.SerializedName
+
+data class WalkingOptionsNavigation(
+    @SerializedName("walking_speed") val walkingSpeed: Double?,
+    @SerializedName("walkway_bias") val walkwayBias: Double?
```

Comment:
We're missing `alleyBias` 👀 https://github.com/mapbox/mapbox-navigation-android/blob/f1e5d047665617c4ef32a3b4b5c9ab29d0ecaa93/libandroid-navigation/src/main/java/com/mapbox/services/android/navigation/v5/navigation/NavigationWalkingOptions.kt#L68

In any case, does this `data class` act as before? 🤔 [`NavigationWalkingOptions`](https://github.com/mapbox/mapbox-navigation-android/blob/master/libandroid-navigation/src/main/java/com/mapbox/services/android/navigation/v5/navigation/NavigationWalkingOptions.kt) is a builder.

Reply:
`WalkingOptionsNavigation` has only 3 params, so don't need to make Builder for it.

Reply:
added `alleyBias `